packages/server/nodemon.json

```diff
@@ -9,5 +9,5 @@
   ],
   "ext": "js,ts,json",
   "ignore": ["src/**/*.spec.ts", "src/**/*.spec.js", "../*/dist/**/*"],
-  "exec": "yarn build && node ./dist/index.js"
+  "exec": "yarn build && node --enable-source-maps ./dist/index.js"
```

Comment:
Do we need this here? This will run only on local dev

Reply:
I put it in because I figure it's also useful in local dev, and the performance impact on local dev is less concerning. Happy to take it out if you don't think it's necessary.